dashboard/src/components/AppView/AppView.tsx

```diff
@@ -100,43 +104,46 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
 
     const kindsWithTable = ["Deployment", "Service", "Secret"];
     const otherResources = manifest
-      .filter(d => kindsWithTable.indexOf(d.kind) < 0)
-      .reduce((acc, r) => {
-        // TODO: skip list resource for now
-        if (r.kind === "List") {
-          return acc;
-        }
-        acc[`${r.kind}/${r.metadata.name}`] = r;
-        return acc;
-      }, {});
+      // TODO: skip list resource for now
+      .filter(d => kindsWithTable.indexOf(d.kind) < 0 && d.kind !== "List");
```

Comment:
It might make sense to use the switch case to also populate the otherResources?

Since the switch case below now it is also used to populate the initial state, it might make sense to use it for otherResources?

Reply:
good point